src/Components/WebAssembly/JSInterop/src/WebAssemblyJSRuntime.cs

```diff
@@ -14,19 +14,19 @@ namespace Microsoft.JSInterop.WebAssembly
     public abstract class WebAssemblyJSRuntime : JSInProcessRuntime
     {
         /// <inheritdoc />
-        protected override string InvokeJS(string identifier, string argsJson)
+        protected override string InvokeJS(string identifier, string argsJson, bool treatReturnAsVoid)
         {
             var noAsyncHandle = default(long);
-            var result = InternalCalls.InvokeJSMarshalled(out var exception, ref noAsyncHandle, identifier, argsJson);
+            var result = InternalCalls.InvokeJSMarshalled(out var exception, ref noAsyncHandle, identifier, argsJson, treatReturnAsVoid);
```

Comment:
It looks like from the comments in the `InvokeJSMarshalled` method that there's some magic that happens here that passes this new `boolean` arg to JS?  Seems like it *should* just work?  Or do we need to be passing the `treatReturnAsVoid` parameter as part of the `argsJson` and somehow extracting it on the JS side?